app/src/main/java/com/conupods/Settings/SettingsPersonalActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.conupods.Settings;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.conupods.OutdoorMaps.View.MapsActivity;
+import com.conupods.R;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SettingsPersonalActivity extends AppCompatActivity {
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.settings_page_personal);
+
+        SharedPreferences preferences = getApplicationContext().getSharedPreferences("Preferences", Context.MODE_PRIVATE);
+        SharedPreferences.Editor prefEdit = preferences.edit();
+
+        Button done = findViewById(R.id.done1);
```

Comment:
declare variables only at the beginning of blocks. (beginning of method in this case)